newsfragments/1571.api.rst

```diff
@@ -0,0 +1 @@
+Fix message name typing, setting it to EntryName to comfort with WorkspaceEntry.
```

Comment:
It's not what the PR does. The PR adds the reason for an entry update.

Reply:
The PR does this, but not just that ;-)
@Sithi5 you should create 2 newsfragments (`1571.api.rst` and `1571.misc.rst`)